_headers.production

```diff
@@ -4,3 +4,10 @@
     X-XSS-Protection: 1; mode=block
```

Comment:
I see the following from the preview:

/
```
wogaa.js:1 Failed to load resource: the server responded with a status of 400 ()
```

/verify
```
Refused to load the script 'https://assets.wogaa.sg/scripts/wogaa.js' because it violates the following Content Security Policy directive: "script-src 'self' 'unsafe-inline' 'unsafe-eval' https://assets.adobedtm.com/". Note that 'script-src-elem' was not explicitly set, so 'script-src' is used as a fallback.
```

/verify looks ok, but not / 🤔

Reply:
I believe it's related to this:
https://github.com/Open-Attestation/verify.gov.sg/blob/efbed6e34cd0a49481226a92941787182b35f093/public/index.html#L47-L51

In development environments (like the deploy preview), should import from `https://assets.dcube.cloud/scripts/wogaa.js` instead.

Can help to check if the deploy preview has the env variable wrongly set to `REACT_APP_WOGAA=production`?